proto/stat.proto

```diff
@@ -213,6 +213,9 @@ message GetStatsRequest {
   // (Optional) The observation period of the observation. If not specified,
   // stats series with any observation period could be returned.
   string observation_period = 6;
+
```

Comment:
This is user-visible change? Does this need any doc update?

Reply:
Yes, this is additional field. I plan to go over the docs after this round of API change (remove payload)